api/migrations/dev-data/data.sql

```diff
@@ -39,15 +39,16 @@ INSERT INTO public."user" (id, state, activationkeyhash, password, createtime, u
 Port Asiaton, MN 44798-0182', '10529', 'New Kaylinfort', '2022-01-23 16:19:10', '2022-01-23 16:19:10', '3b41dca5c568', '3b41dca5c568', NULL, NULL, NULL, NULL, NULL, NULL, false),
-	('e5027d852487', NULL, true, 'J+S', 'J+S', '', '', '', '', '', '2022-01-23 16:19:10', '2022-02-04 21:45:19', '3b41dca5c568', '3b41dca5c568', NULL, NULL, NULL, NULL, NULL, NULL, false),
 	('3c79b99ab424', 'e5027d852487', false, 'Pfila 2023', 'GRGR', 'Grandiose RealityTV Show', 'Adrian Sonderegger', 'Bümplizstrasse 5', '3156', 'Bümpliz', '2023-08-08 07:53:12', '2023-08-08 07:56:11', '9145944210a7', '9145944210a7', 'Abteilung Gryfenberg', 'Pfingstlager', 'Andrea Schneider', NULL, NULL, NULL, false),
 	('6973c230d6b1', 'e5027d852487', false, 'Sola 2023', 'Harry Potter Lager', 'Harry Potter', 'Oskar Hugentobler', 'Rainweg 23', '1999', 'Les Bois', '2023-08-08 09:22:58', '2023-08-08 09:23:39', '9145944210a7', '9145944210a7', 'Pfadi Gryfenberg', 'Sommerlager', 'Andrea Schneider', NULL, NULL, NULL, false),
-	('9c2447aefe38', 'e5027d852487', false, 'Lorem ipsum dolor sit amet', 'Lorem ipsum', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit', 'Lorem ipsum', 'Mainstreet 123', '1234', 'Kongo', '2023-08-12 17:41:55', '2023-08-12 17:45:10', '9145944210a7', '9145944210a7', 'Testinator', 'Summer Tent', 'Max Mustermann', NULL, NULL, NULL, false),
 	('0969e3c95dfc', 'e5027d852487', false, 'Sola 2025', 'Star Wars', 'Die Galaktische Rebellion', NULL, NULL, NULL, NULL, '2023-08-13 06:32:29', '2023-08-13 06:32:29', 'dac7116e02ed', 'dac7116e02ed', NULL, NULL, NULL, NULL, NULL, NULL, false),
-	('70ca971c992f', NULL, false, 'Skilager 2030', 'Auf in die Berge', '', NULL, NULL, NULL, NULL, '2023-09-29 23:24:38', '2023-09-29 23:45:07', 'dac7116e02ed', 'dac7116e02ed', NULL, NULL, NULL, NULL, NULL, NULL, false);
+	('70ca971c992f', NULL, false, 'Skilager 2030', 'Auf in die Berge', '', NULL, NULL, NULL, NULL, '2023-09-29 23:24:38', '2023-09-29 23:45:07', 'dac7116e02ed', 'dac7116e02ed', NULL, NULL, NULL, NULL, NULL, NULL, false),
+	('e5027d852487', NULL, true, NULL, 'J+S', '', '', '', '', '', '2022-01-23 16:19:10', '2022-02-04 21:45:19', '3b41dca5c568', '3b41dca5c568', NULL, NULL, NULL, NULL, NULL, NULL, false),
+	('9c2447aefe38', 'e5027d852487', false, 'Lorem ipsum', 'Lorem ipsum dolor sit amet', 'Lorem ipsum dolor sit amet, consectetur adipiscing elit', 'Lorem ipsum', 'Mainstreet 123', '1234', 'Kongo', '2023-08-12 17:41:55', '2023-08-12 17:45:10', '9145944210a7', '9145944210a7', 'Testinator', 'Summer Tent', 'Max Mustermann', NULL, NULL, NULL, false),
+	('5d28f99890bc', 'e5027d852487', false, '', 'Basiskurs Wolfsstufe 2025', 'Harry Potter', 'Oskar Hugentobler', 'Rainweg 23', '1999', 'Les Bois', '2023-08-08 09:22:58', '2024-09-28 16:14:25', '9145944210a7', '9145944210a7', 'Pfadi Gryfenberg', 'Sommerlager', 'Andrea Schneider', 'JS-CH PBS CH 361-25', 'Basiskurs', 'Sandro Bodruzzi', true);
```

Comment:
```suggestion
	('5d28f99890bc', 'e5027d852487', false, 'PBS CH 361-25', 'Basiskurs Wolfsstufe 2025', 'Harry Potter', 'Oskar Hugentobler', 'Rainweg 23', '1999', 'Les Bois', '2023-08-08 09:22:58', '2024-09-28 16:14:25', '9145944210a7', '9145944210a7', 'Pfadi Gryfenberg', 'Sommerlager', 'Andrea Schneider', 'JS-CH PBS CH 361-25', 'Basiskurs', 'Sandro Bodruzzi', true);
```

I think this would be sensible